gm4_shiny_items/data/gm4_shiny_items/functions/main.mcfunction

```diff
@@ -0,0 +1,22 @@
+
+schedule function gm4_shiny_items:main 16t
+
+# Blacklist Items
+execute at @a as @e[type=item] at @s run function gm4_shiny_items:blacklist_items
```

Comment:
```suggestion
execute at @a as @e[type=item,distance=..30] run function gm4_shiny_items:blacklist_items
```
blacklist_items.mcfunction states that only items within 30 blocks of a player will be executed upon, but that's not the case currently